### ISSUE TITLE
Onis version of the prompt has no completion

### ISSUE THREAD

**Issue by hoschi** [onivim/oni#1574]
In nvim I get completions when I press `Tab` but in Oni I don't:

![2018-02-18-152908_1365x1377_scrot](https://user-images.githubusercontent.com/163128/36352977-9a027638-14c0-11e8-9ea3-e6582304f405.png)

**Comment by Akin909** [comment]
@hoschi do you have the `wildmenu` option set to true? we externalised the wild menu as well so you should be able to have the externalised wildmenu appear although if it isnt you should see the native vim one, depending on what your config setting is it might be there's an incorrect flag somewhere just switched the option from `experimental.wildmenu.mode: true or false` to `wildmenu.mode: true or false`

**Comment by hoschi** [comment]
I used the default config and completely disabled mine. Completion works for available commands:
![2018-02-18-172115_1356x1370_scrot](https://user-images.githubusercontent.com/163128/36354149-218ce6ec-14d0-11e8-95d4-14245d22f0c2.png)

Is that what you meant @Akin909 ?

**Comment by Akin909** [comment]
Yep that's the externalised wildmenu I think its on by default so should just get that unless its disabled in your config
